libs/protocol-ui/src/stories/molecules/DatePicker.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Story } from '@storybook/react';
+import { DatePicker, DatePickerProps } from '../..';
+
+export default {
+    title: 'Components/Molecules/DatePciker',
+    component: DatePicker,
+  
+}
+
+const Template: Story<DatePickerProps> = (args) => <DatePicker {...args} />;
+
+
+export const Default = Template.bind({});
+
+Default.args = {
```

Comment:
Let's try to use an actual example -- can base off our actual DatePicker in the frontend app